docs/design/liqonet_tunnelEndpoint.md

```diff
@@ -0,0 +1,17 @@
+# TunnelEndpoint-Operator
+## Overview
+This component is in charge of establishing a VPN tunnel with the peering clusters, which is created between the (local) Gateway Node and the Gateway Node of the remote cluster.
+All the traffic between the local and remote cluster is first delivered to the Gateway Node, then tunneled towards the remote gateway and finally delivered to the desired destination.
+The traffic can leave the local cluster as is in case the home and remote addressing spaces do not overlap; otherwise, the traffic crosses a properly configurated NAT in order to avoid overlapped spaces.
```

Comment:
Who configures the NAT?

Reply:
The iptables rules configuring the NAT are managed by the routeOperator.